src/main/kotlin/no/nav/syfo/rules/ValidationRuleChain.kt

```diff
@@ -67,7 +67,9 @@ enum class ValidationRuleChain(
             infotrygdForesp.sMhistorikk?.sykmelding != null && infotrygdForesp.sMhistorikk.sykmelding
                     .any { infotrygdForespSykmelding ->
                         infotrygdForespSykmelding.periode?.arbufoerFOM != null && (sykmeldingPeriods.fom.isBefore(infotrygdForespSykmelding.periode.arbufoerFOM) ||
-                                sykmeldingPeriods.fom.isEqual(infotrygdForespSykmelding.periode.arbufoerFOM))
+                                sykmeldingPeriods.fom.isEqual(infotrygdForespSykmelding.periode.arbufoerFOM)) &&
```

Comment:
Det hadde vært veldig fint om vi konverterte `infotrygdForespSykmelding` om til noe mer lesbart før vi drar den inn i domenet vårt. Måtte tenke litt før jeg forstod hva arbufoerFOM var for noe.

Reply:
enig, tenker vi tar det i egen pr, å lage oppgave for det.